entrypoint.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+set -e
+while getopts "a:b:c:d:e:f:g:h:i:j:" o; do
+   case "${o}" in
+       a)
+         export scanType=${OPTARG}
+       ;;
+       b)
+         export format=${OPTARG}
+       ;;
+       c)
+         export template=${OPTARG}
+       ;;
+       d)
+         export exitCode=${OPTARG}
+       ;;
+       e)
+         export ignoreUnfixed=${OPTARG}
+       ;;
+       f)
+         export vulnType=${OPTARG}
+       ;;
+       g)
+         export severity=${OPTARG}
+       ;;
+       h)
+         export output=${OPTARG}
+       ;;
+       i)
+         export imageRef=${OPTARG}
+       ;;
+       j)
+         export scanRef=${OPTARG}
+       ;;
+  esac
+done
+
+scanType=$(echo $scanType | tr -d '\r')
+export artifactRef="${imageRef}"
+if [ "${scanType}" = "fs" ];then
+  artifactRef=$(echo $scanRef | tr -d '\r')
+fi
+
+ARGS=""
+if [ $format ];then
+ ARGS="$ARGS --format $format"
+fi
+if [ $template ] ;then
+ ARGS="$ARGS --template $template"
+fi
+if [ $exitCode ];then
+ ARGS="$ARGS --exit-code $exitCode"
+fi
+if [ "$ignoreUnfixed" == "true" ];then
+  ARGS="$ARGS --ignore-unfixed"
+fi
+if [ $vulnType ];then
+  ARGS="$ARGS --vuln-type $vulnType"
+fi
+if [ $severity ];then
+  ARGS="$ARGS --severity $severity"
+fi
+if [ $output ];then
+  ARGS="$ARGS --output $output"
+fi
+
```

Comment:
Is there a way to show the actual trivy command that gets run below for debugging purposes? Currently I see something like this which isn't helpful
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/1254783/108413813-535aba80-71e0-11eb-88a4-e9dcb1c09388.png">

Reply:
Just added the echo statement, rebuilding the job now to show options & args.

Reply:
<img width="1065" alt="Screenshot 2021-02-19 at 1 36 51 AM" src="https://user-images.githubusercontent.com/18293335/108415048-fb10d080-7252-11eb-88de-0a45c57f1e4d.png">

ImageScan: https://github.com/rahul2393/trivy-ci-test/runs/1930144696